src/lib/core/ripple/ripple.ts

```diff
@@ -181,5 +176,28 @@ export class MatRipple implements OnInit, OnDestroy, RippleTarget {
       this._rippleRenderer.setupTriggerEvents(this.trigger);
     }
   }
+
+  /**
+   * Launches a manual ripple using the specified ripple configuration.
+   * @param config Configuration for the manual ripple.
+   */
+  launch(config: RippleConfig): RippleRef;
```

Comment:
There should be a unit test that just passes in a config

Reply:
Good idea. Done